src/bos/operators/utils/boot_image_metadata/__init__.py

```diff
@@ -66,6 +66,12 @@ def rootfs(self):
         """
         return None
 
+    @property
```

Comment:
I feel like the class BootImageMetaData should have been designated an abstract class if it wasn't going to fill in all of the attributes. Not your fault in this PR; just making an observation.

Reply:
I agree with you 100%